pages/mdakits.md

```diff
@@ -1,21 +1,32 @@
 ---
 layout: page
-title: Used By
+title: MDAKits and other tools using MDAnalysis
 ---
 
-While our documentation is mostly focused on using MDAnalysis for
-exploratory analysis it is equally well suited to build your own
-analysis library on top of it. Below is a list of projects we know
-about that use MDAnalysis.
+MDAnalysis is developed with extensibility in mind, allowing
+scientists to create various tools with the components
+that the library provides. To help developers in this process,
+and to advertise these wonderful tools to the MDAnalysis user community,
+we have created an ecosystem of [MDAnalysis Toolkits](https://mdakits.mdanalysis.org/about.html).
 
 
-*If you know of other tools that use MDAnalysis please share
-them with us on [Twitter]({{ site.twitter.url }}) or on our
-[mailinglist]({{ site.mailinglists.discussion.url }}) so that we
-can include them here* or open a pull request by [editing the file `used-by.md`](https://github.com/MDAnalysis/MDAnalysis.github.io/edit/master/pages/used-by.md).
+Please look at the [registry of toolkits](https://mdakits.mdanalysis.org/mdakits.html)
+for a list of toolkits that [meet the minimum MDAKit requirements](insert link here).
+These are regularly tested against the latest and development versions of
+MDAnalysis, so if you see a green badge it should be ok to use!
```

Comment:
```suggestion
MDAnalysis, so if you see a green badge, we expect that it should be good to use!
```

"Ok" felt a little unenthusiastic -- minor comment, but swap for "good"?

Reply:
🤔 We probably should be even clearer that it's not on us but on the MDAKit authors, I might try something else.

Reply:
Updated